src/directives/formly-field.test.js

```diff
@@ -997,6 +997,28 @@ describe('formly-field', function() {
       expect(() => compileAndDigest()).to.throw();
     });
 
+    it(`should share the form with a fieldGroup`, () => {
```

Comment:
I think you put this in the `describe` for `model`. Do you mind creating a new `describe('fieldGroup')` and putting this test in that one? While you're at it, you could move the `describe(fieldGroup with specified "key" property)` inside that new one if you want. :+1: